rotkehlchen/chain/evm/decoding/aura_finance/utils.py

```diff
@@ -0,0 +1,153 @@
+import logging
+from typing import TYPE_CHECKING, Any
+
+from eth_utils import to_checksum_address
+
+from rotkehlchen.assets.asset import Asset, UnderlyingToken
+from rotkehlchen.assets.utils import TokenEncounterInfo, get_or_create_evm_token
+from rotkehlchen.chain.evm.contracts import EvmContract
+from rotkehlchen.chain.evm.decoding.utils import update_cached_vaults
+from rotkehlchen.constants import ONE
+from rotkehlchen.constants.prices import ZERO_PRICE
+from rotkehlchen.constants.resolver import evm_address_to_identifier
+from rotkehlchen.errors.misc import RemoteError
+from rotkehlchen.globaldb.handler import GlobalDBHandler
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+from rotkehlchen.types import CacheType, EvmTokenKind, Price
+
+from .constants import AURA_BOOSTER_ABI, CHAIN_ID_TO_BOOSTER_ADDRESSES, CPT_AURA_FINANCE
+
+if TYPE_CHECKING:
+    from rotkehlchen.assets.asset import EvmToken
+    from rotkehlchen.chain.evm.node_inquirer import EvmNodeInquirer
+    from rotkehlchen.db.dbhandler import DBHandler
+    from rotkehlchen.inquirer import Inquirer
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+
+def query_aura_pools(evm_inquirer: 'EvmNodeInquirer') -> None:
+    update_cached_vaults(
+        display_name='Aura Finance',
+        database=evm_inquirer.database,
+        query_vaults=lambda: _query_aura_pools(evm_inquirer),
+        process_vault=_process_aura_pool,
+        cache_key=(CacheType.AURA_POOLS, str(evm_inquirer.chain_id.value)),
+    )
+
+
+def _process_aura_pool(database: 'DBHandler', vault: dict[str, Any]) -> None:
+    """Store Aura pool data.
+
+    May raise:
+    - NotERC20Conformant
+    """
+    chain_id = vault['chain_id']
+    aura_pool_token = get_or_create_evm_token(
+        userdb=database,
+        evm_address=vault['aura_pool_token'],
+        chain_id=chain_id,
+        encounter=TokenEncounterInfo(
+            description=f'Querying {chain_id.to_name()} aura finance balances',
```

Comment:
in this function you are repeating chain_id.to_name() 3 times. Put this out in a variable and reuse it